src/recommendations/ProductCard/index.jsx

```diff
@@ -75,11 +74,12 @@ const ProductCard = ({
     ],
   );
   const handleCardClick = () => {
-    trackRecommendationCardClickOptimizely(userId?.toString());
     trackRecommendationsClicked(
-      product.courseKey,
-      false,
+      product.uuid,
+      product.cardType,
+      product.productSource.name,
       position + 1,
+      false,
```

Comment:
[Confirming] isControl=false means user is seeing the recs?

Reply:
Yes.